### PR TITLE
Fix output for decimal values

### PR DESCRIPTION
**Pull request by vvysotskyi**
- Output before the changes:
These rows are not expected (first 10):
I��

- Output after the changes:
These rows are not expected (first 10):
-1E-29

### CODE REVIEW

**Comment by vvysotskyi** [comment]
@Agirish, @kkhatua Could you please take a look?

File: framework/src/main/java/org/apache/drill/test/framework/Utils.java [hunk 1]
```diff
@@ -538,7 +520,39 @@ public static String getSqlResult(ResultSet resultSet) throws SQLException {
...
+    int columnCount = resultSet.getMetaData().getColumnCount();
+    List<Object> values = new ArrayList<>();
+    for (int i = 1; i <= columnCount; i++) {
+      try {
+        switch (resultSet.getMetaData().getColumnType(i)) {
+          case Types.BINARY:
+          case Types.VARBINARY:
+            byte[] bytes = resultSet.getBytes(i);
+            if (bytes != null) {
+              values.add(new String(bytes, StandardCharsets.UTF_8.name()));
```
Comment (kkhatua) [inline framework/src/main/java/org/apache/drill/test/framework/Utils.java]: Don't we need to handle UTF-16 character set and Date objects?

**Comment by Agirish** [comment]
@cchang738, can you also kindly take a look?

File: framework/src/main/java/org/apache/drill/test/framework/Utils.java (hunk 1 above)
Comment (cchang738) [reply framework/src/main/java/org/apache/drill/test/framework/Utils.java]: This looks good to me and good improvement. Make sure you run all tests locally to confirm.
+1

Comment (vvysotskyi) [reply framework/src/main/java/org/apache/drill/test/framework/Utils.java]: @cchang738, it was checked with pre-commit tests.

**Comment by Agirish** [comment]
+1. Looks good; thanks for making these changes, @vvysotskyi 
I'll see if I can run this through tests and will have this merged.

**Pull request merged**